desktop/main.ts

```diff
@@ -310,7 +311,36 @@ const mainWindow = (): Promise<void> => {
                 });
 
                 ipcMain.handle(ELECTRON_EVENTS.REQUEST_DEVICE_ID, () => machineId());
-
+                ipcMain.handle(ELECTRON_EVENTS.OPEN_LOCATION_SETTING, () => {
+                    return new Promise((resolve, reject) => {
+                        let command = '';
+
+                        switch (process.platform) {
+                            case 'darwin':
+                                command = 'open x-apple.systempreferences:com.apple.preference.security?Privacy_Location';
+                                break;
+                            case 'win32':
+                                command = 'start ms-settings:privacy-location';
+                                break;
+                            case 'linux':
+                                command = 'gnome-control-center';
+                                break;
```

Comment:
AFAIK we only provide the desktop app on macOS.

Reply:
done